src/main/java/com/faforever/client/game/TeamCardController.java

```diff
@@ -83,6 +90,17 @@ public void setPlayersInTeam(String team, List<Player> playerList, Function<Play
     teamNameLabel.setText(teamTitle);
   }
 
+  public void showRatingChange(ObservableMap<String, List<PlayerStats>> teams) {
+    teams.values().
+        forEach(playerStatsList ->
+            playerStatsList.forEach(playerStats -> {
+              if (ratingChange.containsKey(playerStats.getPlayerId())) {
+                ratingChange.get(playerStats.getPlayerId()).setRatingChange(playerStats);
+              }
+            })
+        );
```

Comment:
* Expect `Map` instead of `ObservableMap` as parameter
* Period goes on same line as method call: `.forEach`
* Use `filter()` and `flatMap()` for cleaner code
* Stop looping after result has been found

```
teams.values()
  .flatMap(List::stream)
  .filter(playerStats -> ratingChangeLabelControllerByPlayerId.containsKey(playerStats.getPlayerId()))
  .findFirst()
  .map(playerStats -> ratingChangeLabelControllerByPlayerId.get(playerStats.getPlayerId()))
  .ifPresent(PlayerStats::setRatingChange);
```

Reply:
I did 
```

    teams.values()
        .stream()
        .flatMap(List::stream)
        .filter(playerStats -> ratingChangeControllersByPlayerId.containsKey(playerStats.getPlayerId()))
        .forEach(playerStats -> ratingChangeControllersByPlayerId.get(playerStats.getPlayerId()).setRatingChange(playerStats));
```
because some things can not be realized from your example:

1. PlayerStats::setRatingChange is not a static Method 
2.   .findFirst() all labels should be set not only one